server/src/pages/shot/editor.js

```diff
@@ -140,3 +146,54 @@ exports.Editor = class Editor extends React.Component {
     this.drawContext.stroke();
   }
 }
+
+class ColorPicker extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      picker: 'inactive',
+      color: '#000'
+    };
+  }
+
+  render() {
+    let colorBoard;
+    if (this.state.picker == 'active') {
+      colorBoard = <div className="color-board">
+        <div className="row">
```

Comment:
This is a big logic chunk... I think this should be done in a later method, and included like `{this.state.pickerActive ? this.renderColorBoard : null}`